ci/github/collect_troubleshooting

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euxo pipefail
+collect_troubleshooting () {
```

Comment:
Why do we need this? We already have `collect_troubleshooting` in `create_infra`.